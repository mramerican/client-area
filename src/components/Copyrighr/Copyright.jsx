import React from 'react';
import styles from './Copyright.module.scss';

const Copyright = () =>
  <div className={styles.wrapper}>
    Evoplay &#169; 2023.<br />
    All rights reserved.
  </div>

export default Copyright;