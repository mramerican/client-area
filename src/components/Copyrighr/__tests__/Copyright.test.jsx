import { render } from '@testing-library/react';

import Copyright from 'components/Copyrighr/Copyright';

describe('Copyright', () => {
  test('initial', () => {
    const { getByText, asFragment } = render(<Copyright />);
    const linkElement = getByText(/Evoplay/i);

    expect(linkElement).toBeInTheDocument();
    expect(asFragment(<Copyright />)).toMatchSnapshot();
  });
});
