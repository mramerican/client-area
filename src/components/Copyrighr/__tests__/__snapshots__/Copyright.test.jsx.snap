// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright initial 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    Evoplay Â© 2023.
    <br />
    All rights reserved.
  </div>
</DocumentFragment>
`;
